src/components/victory-label.jsx

```diff
@@ -106,96 +81,41 @@ export default class VictoryLabel extends React.Component {
      */
     y: PropTypes.number,
     /**
-     * The dx prop defines a horizontal shift from the `x` coordinate.
+     * The dx prop defines a horizontal shift from the `x` coordinate in pixels.
      */
     dx: PropTypes.oneOfType([
       PropTypes.number,
-      PropTypes.string,
       PropTypes.func
     ]),
     /**
-     * The dy prop defines a vertical shift from the `y` coordinate. Since this
-     * component already accounts for `capHeight`, `lineHeight`, and
-     * `verticalAnchor`, this will usually not be necessary.
+     * The dy prop defines a vertical shift from the `y` coordinate. in px
      */
     dy: PropTypes.oneOfType([
       PropTypes.number,
-      PropTypes.string,
       PropTypes.func
     ])
   };
 
-  static defaultProps = {
-    capHeight: "0.71em", // Magic number from d3.
-    lineHeight: 1
-  };
-
-  getStyles(props) {
-    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
-    return Chart.evaluateStyle(style);
-  }
-
-  getHeight(props, type) {
-    const height = Chart.evaluateProp(props[type]);
-    return typeof height === "number" ? `${height}em` : height;
-  }
-
-  getContent(props) {
-    if (props.children) {
-      const child = Chart.evaluateProp(props.children);
-      return `${child}`.split("\n");
-    }
-    return [""];
-  }
-
-  getDy(props, content, lineHeight) {
-    const dy = props.dy ? Chart.evaluateProp(props.dy) : 0;
-    const length = content.length;
-    const capHeight = this.getHeight(props, "capHeight");
-    const verticalAnchor = props.verticalAnchor ?
-      Chart.evaluateProp(props.verticalAnchor) : "middle";
-    switch (verticalAnchor) {
-    case "end":
-      return Style.calc(
-        `${dy} +  ${capHeight} / 2 + (0.5 - ${length}) * ${lineHeight}`
-      );
-    case "middle":
-      return Style.calc(
-        `${dy} + ${capHeight} / 2 + (0.5 - ${length} / 2) * ${lineHeight}`
-      );
-    default:
-      return Style.calc(`${dy} + ${capHeight} / 2 + ${lineHeight} / 2`);
-    }
-  }
-
   render() {
-    const lineHeight = this.getHeight(this.props, "lineHeight");
-    const transform =
-      this.props.transform && Style.toTransformString(Chart.evaluateProp(this.props.transform));
     const textAnchor = this.props.textAnchor ?
       Chart.evaluateProp(this.props.textAnchor) : "start";
```

Comment:
What's the value of running all of these through evaluateProp? Since there is no second argument provided, functional props will not be passed anything, therefore it seems like there would be no reason to use functional props for these.